include/mysql/components/services/psi_thread_bits.h

```diff
@@ -283,7 +283,7 @@ typedef void (*get_thread_event_id_v1_t)(unsigned long long *thread_internal_id,
                                          unsigned long long *event_id);
 
 /* Duplicate definitions to avoid dependency on mysql_com.h */
-#define PSI_USERNAME_LENGTH (32 * 3)
+#define PSI_USERNAME_LENGTH (80 * 3)
```

Comment:
Shouldn't this be `80 * 4` as in 8.0 default charset is `utf8mb4`?

Reply:
All the system tables are still utf8mb3